modules/dns/variables.tf

```
@@ -49,3 +49,15 @@ variable "dns_record_set" {
     error_message = "The specified DNS record type is not valid."
   }
 }
+
+variable "base64_encoded_dns_records_file" {
+  type        = string
+  description = "The base64 encoded DNS zone file in BIND format that contains the details of the DNS records. For more information about the import format, refer https://cloud.ibm.com/docs/dns-svcs?topic=dns-svcs-managing-dns-records&interface=ui. Required if dns_records_file is not passed."
```

```suggestion
  description = "The base64-encoded DNS zone file in BIND format that contains the details of the DNS records [Learn more](https://cloud.ibm.com/docs/dns-svcs?topic=dns-svcs-managing-dns-records&interface=ui). Required if `dns_records_file` is not specified."
```

Could use the markdown link format and `Learn more` for all links to docs. They should render correctly in the catalog UI.